python/cuspatial/cuspatial/core/spatial/join.py

```diff
@@ -342,3 +323,40 @@ def quadtree_point_to_nearest_linestring(
             linestring_points_y,
         )
     )
+
+
+@cuda.jit
+def binarize(in_col, out, width):
```

Comment:
Whoah. Shoud we move this stuff to libcuspatial?

Reply:
Does this benefit cuSpatial? I don't see a usage of this other than converting the result of Point in Polygon to dataframe.

Reply:
Doing it faster benefits cuSpatial users. Generally Numba and Python add a lot of overhead.

Reply:
Can probably punt to a future optimization. Another question is whether we should change the point in polygon interface, or simply leave this as a cuda kernel bind to python.